src/Uno.Foundation/Uno.Core.Extensions/Uno.Core.Extensions/StringExtensions.cs

```diff
@@ -246,19 +129,7 @@ public static string TrimStart(this string source, string trimText)
 		{
 			if (!string.IsNullOrEmpty(trimText) && source.StartsWith(trimText, StringComparison.Ordinal))
 			{
-				return source.Substring(trimText.Length).TrimStart(trimText, StringComparison.Ordinal);
-			}
-			else
-			{
-				return source;
-			}
-		}
-
-		public static string TrimStart(this string source, string trimText, StringComparison comparisonType)
-		{
-			if (!string.IsNullOrEmpty(trimText) && source.StartsWith(trimText, comparisonType))
-			{
-				return source.Substring(trimText.Length).TrimStart(trimText, comparisonType);
+				return source.Substring(trimText.Length);
```

Comment:
Previously we were keep calling `TrimStart` until no matches are found. This is never the behavior needed by any of the callers.